src/vs/workbench/browser/parts/editor/media/titlecontrol.css

```diff
@@ -30,7 +30,7 @@
 }
 
 .monaco-workbench .part.editor > .content .editor-group-container > .title .monaco-icon-label::before {
-	height: 35px; /* tweak the icon size of the editor labels when icons are enabled */
+	height: var(--tab-height); /* tweak the icon size of the editor labels when icons are enabled */
```

Comment:
This CSS file is shared between `tabsTitleControl.ts` and `noTabsTitleControl.ts`, so we would need make sure the variable is defined also from `noTabsTitleControl.ts`. 

This goes into the direction of my other comment: we could just support this setting for no tabs as well.